src/builtin/builtin.go

```diff
@@ -0,0 +1,9 @@
+package builtin
+
+// Check that builtin types are correctly graphed.
```

Comment:
add to comment: "this simulates the real builtin package and makes sure we properly handle references to builtin defs."